service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/ControlledAzureDatabaseResource.java

```diff
@@ -158,7 +147,7 @@ public void addCreateSteps(
 
   @VisibleForTesting
   List<Step> getAddSteps(FlightBeanBag flightBeanBag) {
-    var steps = new ArrayList<Step>();
+    ArrayList<Step> steps = new ArrayList<>();
```

Comment:
nit: mostly a style issue, but I'm fine using var for cases like these where the type is clear from the declaration or like below where it's clear from the var name